ODLA/platforms/dnnl/odla_dnnl.cc

```diff
@@ -480,8 +489,14 @@ odla_status odla_BindToOutput(odla_value value, odla_void* data_ptr,
 odla_status odla_BindToOutputById(const odla_value_id value_id,
                                   odla_void* data_ptr, odla_context context) {
   std::string name((const char*)value_id);
+  auto& outputs_v = context->comp->outputs_v;
   auto val = context->comp->outputs[name];
-  return odla_BindToOutput(val, data_ptr, context);
+  if (outputs_v.find(name) != outputs_v.end()) {
+    outputs_v[name].second = data_ptr;
```

Comment:
how about just "outputs_v[name] = { val, data_ptr} " ?
or we can assert if duplicated binding happens.

Reply:
got it